src/lib/hardforks.ts

```diff
@@ -0,0 +1,42 @@
+import { CURRENT_MAINNET_HARDFORK } from './constants';
+
+export enum Hardforks {
```

Comment:
We might want to call this MainnetHardforks and have it live in `src/chains/mainnet/hardforks.ts`. The rationale here is some chains chains will have their own sets of hardforks too